### PR TITLE
Release Notes for v0.3.0

### PR DESCRIPTION
**Pull request by trbehera**
Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

**Review by TomBarnes — COMMENTED** [review]
Line 7 seems wrong. There should be some sort of high-level summary of what's implementa and what is not.

File: release-notes-v0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+v0.3.0
+
+### New features
+
+This release contains a reference implementation of [FIDO IoT draft spec](https://fidoalliance.org/specs/fidoiot/FIDO-IoT-spec-v1.0-wd-20200730.html). This implementation inherits the initial code from the protocol-next branch of [PRI](https://github.com/secure-device-onboard/pri) repository.
+
+* The core protocol implementation supports "list of crypto algorithms", "list of key-exchange protocols".
+* Protocol-samples are added to help developers validate core protocol implementation. These samples
```
Comment (TomBarnes) [inline release-notes-v0.3.0.md]: Suggest "Protocol-samples are provided"

File: release-notes-v0.3.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Discontinued features
+
+None  
+
+### Fixed Issues
+
+None
+
+### Known Issues
```
Comment (KiranSukhavasi) [inline release-notes-v0.3.0.md:28]: If possible Please include the below info in the known issues,

* [PRI-FIDO][Protocol   Sample] Warnings when \mvn exec:java\ command is executed for protocol sample   components can be ignored.
* [Owner] Need to mention about service info transfers more than 14 MB size that currently we are not supporting.
* Resale/reuse for Client Intel is not supported for this release.

**Review by KiranSukhavasi — CHANGES_REQUESTED** [review]

File: release-notes-v0.3.0.md (hunk 1 above)
Comment (trbehera) [reply release-notes-v0.3.0.md]: Updated.

File: release-notes-v0.3.0.md (hunk 2 above)
Comment (trbehera) [reply release-notes-v0.3.0.md:28]: Issues related to mvn exec -> Documented in https://github.com/secure-device-onboard/pri-fidoiot/README.md

Issue regarding serviceinfo file upload -> Updated

Comment (KiranSukhavasi) [reply release-notes-v0.3.0.md:28]: @trbehera , Can we also say that Client-Intel suppports only EPID not On-Die / ECDSA.

Comment (trbehera) [reply release-notes-v0.3.0.md:28]: This is not a supported opensource platform.

**Review by v-vasavi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Pull request merged**